spec/rails_helper.rb

```diff
@@ -0,0 +1,64 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```

Comment:
I see opted for using `RSpec`.

Do you mind then nuking all that was in the `test` directory?

When a new rails project is created, it comes with minitest out-of-the-box. And It'd not make sense keeping it

Reply:
Yes , we will delete this